### ISSUE TITLE
⛔ 🧩 🛟 Hello Child Theme not displaying PUBLISHED pages

### ISSUE THREAD

**Issue by EvanTreacy** [elementor/elementor#28545]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Hi,

I am experiencing the following error with Hello Child Theme activated and after 'Publishing' pages.
When I create a new page and begin editing with elementor Pro, the builder loads as expected and I can design and publish my page. After 'Publishing' the page and viewing the page url, the Elementor page is not showing the contents, and Elementor Builder will not load.

Troubleshooting:
Disabled all plugins (except Elementor, Elementor Pro) and issue still persists.
I changed theme to Default theme (WordPress 2024) and issue was fixed. Hello Parent theme also fixes the issue. 
The issue only occurs on Hello Child Theme.
Caching is disabled.

I have discovered that changing the status of a page from 'Published' to 'Draft' fixes the issue. On Drafts I can load the page url and it will display Elementor page that I designed/built, and Elementor Page Builder will load like normal.

 Changing 'Permalink' setting to 'Plain' also fixes this. But, I must use 'Post-name' permalink.

### Steps to reproduce

1. Activate Hello Child Theme.
2. Create new page.
3. Edit with Elementor Pro, and Publish.

### Expected behavior

After Publishing:
1. Page url is not loading Elementor Page previously built prior to Publishing.
2. Cannot access Elementor Pro builder for the page.

Changing status of page from 'Published' to 'Draft' allows me to view elementor page url correctly, and I can open Elementor Builder on that page.

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [ ] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt


== Server Environment ==
	Operating System: Linux
	Software: LiteSpeed
	MySQL version: MySQL Community Server - GPL v8.0.39
	PHP Version: 8.1.29
	PHP Memory Limit: 256M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 512M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.6.1
	Site URL: https://asgatechgroup.com
	Home URL: https://asgatechgroup.com
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: 0
	Debug Mode: Active

== Theme ==
	Name: AsgaTech Group Child
	Version: 2.0.0
	Author: Evan Treacy
	Child Theme: Yes
	Parent Theme Name: AsgaTech Group
	Parent Theme Version: 3.1.0
	Parent Theme Author: Evan Treacy

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0

== Active Plugins ==
	Advanced Custom Fields PRO
		Version: 6.3.5
		Author: WP Engine

	All In One WP Security
		Version: 5.3.2
		Author: All In One WP Security & Firewall Team

	AnyWhere Elementor Pro (Premium)
		Version: 2.27
		Author: WP Vibes

	Ele Custom Skin
		Version: 3.1.9
		Author: Dudaster.com

	Elementor
		Version: 3.23.4
		Author: Elementor.com

	Elementor Pro
		Version: 3.23.3
		Author: Elementor.com

	JetSmartFilters
		Version: 3.5.4
		Author: Crocoblock

	LiteSpeed Cache
		Version: 6.4.1
		Author: LiteSpeed Technologies

	Rank Math SEO
		Version: 1.0.226
		Author: Rank Math

	Rank Math SEO PRO
		Version: 3.0.69
		Author: Rank Math

	Really Simple SSL
		Version: 8.3.0.1
		Author: Really Simple Plugins

	Temporary Login
		Version: 1.2.0
		Author: Elementor.com

	TranslatePress - Multilingual
		Version: 2.8.3
		Author: Cozmoslabs, Razvan Mocanu, Madalin Ungureanu, Cristophor Hurduban

	UpdraftPlus - Backup/Restore
		Version: 1.24.4
		Author: UpdraftPlus.Com, DavidAnderson

	WPForms
		Version: 1.9.0.4
		Author: WPForms


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	
	recaptcha: Active


== Elementor Experiments ==
	Improved CSS Loading: Active by default
	Inline Font Icons: Active by default
	Additional Custom Breakpoints: Active by default
	Flexbox Container: Active by default
	Grid Container: Active by default
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Control Loading: Active by default
	Optimized Markup: Inactive by default
	Plugin Onboarding: Active by default
	Default to New Theme Builder: Active by default
	Header & Footer: Active by default
	Elementor Home Screen: Active by default
	Editor Top Bar: Inactive by default
	Build with AI: Active by default
	Landing Pages: Inactive by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Active by default
	Element Caching: Inactive by default
	Link In Bio: Inactive by default
	Floating Buttons: Inactive by default
	Elementor Editor Events: Inactive by default
	Display Conditions: Active by default
	Form Submissions: Active by default
	Menu: Inactive by default
	Taxonomy Filter: Active by default
	Search: Inactive by default


== Log ==
	
PHP: showing 5 of 5PHP: 2024-08-23 13:30:10 [warning X 472][../wp-content/plugins/elementor/includes/conditions.php::87] Undefined array key "background_color" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-08-26 09:22:53 [error X 9][../wp-content/plugins/elementor/includes/preview.php::209] Uncaught Error: Call to a member function get_container_attributes() on bool in ../wp-content/plugins/elementor/includes/preview.php:209
Stack trace:
#0 ../wp-content/plugins/elementor-pro/modules/theme-builder/documents/theme-document.php(170): Elementor\Preview->builder_wrapper()
#1 ../wp-content/plugins/elementor-pro/modules/theme-builder/classes/locations-manager.php(364): ElementorPro\Modules\ThemeBuilder\Documents\Theme_Document->print_content()
#2 ../wp-content/plugins/elementor-pro/modules/theme-builder/api.php(13): ElementorPro\Modules\ThemeBuilder\Classes\Locations_Manager->do_location()
#3 ../wp-content/themes/asgatech-group-child/header.php(56): elementor_theme_do_location()
#4 ../wp-includes/template.php(810): require_once('/home/asgatech/...')
#5 ../wp-includes/template.php(745): load_template()
#6 ../wp-includes/general-template.php(48): locate_template()
#7 ../wp-content/themes/asgatech-group/index.php(15): get_header()
#8 ../wp-includes/template-loader.php(106): include('/home/asgatech/...')
#9 ../wp-blog-header.php(19): require_once('/home/asgatech/...')
#10 /home/asgatech/public_html/index.php(17): require('/home/asgatech/...')
#11 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-08-26 13:46:09 [warning X 14][../wp-content/plugins/elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-08-26 14:04:00 [warning X 11][../wp-content/plugins/elementor-pro/modules/dynamic-tags/acf/tags/acf-image.php::40] Undefined array key 1 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-08-27 15:41:58 [warning X 1][../wp-content/plugins/elementor/core/files/css/base.php::783] Undefined array key "padding_mobile" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 9 of 9JS: 2024-08-23 13:30:27 [error X 5][../wp-content/plugins/elementor/assets/js/editor-modules.min.js?ver=3.23.4:3:5491] r.get is not a function 
JS: 2024-08-23 13:31:32 [error X 31][../wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.23.3:3:66072] can't convert undefined to object 
JS: 2024-08-26 10:27:05 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.23.4:3:54579] V is undefined 
JS: 2024-08-26 14:02:38 [error X 1][../wp-content/plugins/elementor/assets/js/frontend.min.js?ver=3.23.4:2:30732] elementor.settings.page is undefined 
JS: 2024-08-26 15:19:29 [error X 4][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.23.4:3:832928] this.store.find(...) is undefined 
JS: 2024-08-27 08:25:22 [error X 2][../wp-content/plugins/elementor/assets/js/admin.min.js?ver=3.23.4:3:52333] c[0] is undefined 
JS: 2024-08-27 12:56:35 [error X 1][https://editor-static-bucket.elementor.com/ai-integration.js?v=1724763366559&ver=3.23.4:8:48] n.config is undefined 
JS: 2024-08-27 15:40:09 [error X 4][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.23.4:3:1264968] $ is null 
JS: 2024-08-27 16:06:38 [error X 1][../wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.23.4:3:656177] B is undefined 



== Elementor - Compatibility Tag ==
	
	AnyWhere Elementor Pro (Premium): Compatibility not specified
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by EvanTreacy** [comment]
Error Solved.

I can confirm that this was not a bug caused by Hello Theme or Elementor.

For anyone who may come across this error, be aware not to use any reserved terms when creating CPT's in your child theme.

The issue was caused by using a Term reserved by WordPress in one of my CPT's in my Child Theme. Specifically, I used the taxonomy 'year' which caused conflicts with core functionality.

To fix this I simply changed 'year' to something else: 'publishing-year'.
